doc/openocd.texi

```diff
@@ -11524,6 +11524,20 @@ as in the mie CSR (defined in the RISC-V Privileged Spec).
 For details on this trigger type, see the RISC-V Debug Specification.
 @end deffn
 
+@deffn {Command} {riscv reserve_trigger} [index [@option{on|off}]]
+Manages the set of reserved triggers. Reserving a trigger results in OpenOCD
+not using it internally (e.g. skipping it when setting a watchpoint or a
+hardware breakpoint), so that the user or the application being run has
+unfettered control over the trigger. By default there are no reserved triggers.
+
+@enumerate
+@item @var{index} specifies the index of a trigger to reserve or free up.
+@item The second argument specifies whether the trigger should be reserved.
```

Comment:
I have tried to make the command documentation even clearer:

```suggestion
@item The second argument specifies whether the trigger should be reserved (@var{on}) or a prior reservation cancelled (@var{off}).
```

Reply:
Addressed.